infrastructure/haproxy_reload.sh

```diff
@@ -50,10 +53,11 @@ replace() {
 configure() {
   instance=$1
   instance_prefix="${instance}_prefix"
-  real_port="${!instance_prefix}${real}"
+  http_port="${!instance_prefix}${http}"
   stats_port="${!instance_prefix}${stats}"
   export PORT_STATS=${stats_port}
-  export PORT_HTTP=${real_port}
+  export PORT_HTTP=${http_port}
+  [[ ${HAPROXY_SSL} == "true" ]] && export CERT_OPTS="ssl crt /etc/haproxy/haproxy.pem"
```

Comment:
I would move it out form here to `generate_yml.sh`,
so `CERT_OPTS` would be defined by user - he might want to have different place for certs.
(default can be `ssl crt /etc/haproxy/haproxy.pem`)
sth like:
```
HAPROXY_CERT_OPTS=${HAPROXY_CERT_OPTS:-"ssl crt /etc/haproxy/haproxy.pem"}
```
you also would have to add it to `docker-compose.yml.tpl`